src/backend/opencl/kernel/mean.hpp

```diff
@@ -340,7 +341,8 @@ void mean_first_launcher(Param out, Param owt,
         if (output_weight) { options << " -D OUTPUT_WEIGHT"; }
 
         if (std::is_same<Ti, double>::value ||
-                std::is_same<Ti, cdouble>::value) {
+                std::is_same<Ti, cdouble>::value ||
+                std::is_same<To, cdouble>::value) {
```

Comment:
Did you mean to use cdouble here or double ?

Reply:
argh, yes...